W18_proj00.md

```diff
@@ -0,0 +1 @@
+# Claudia Zeng Kerry Mo W18 Proj00
```

Comment:
You're missing the rest of this file... You can skip the "evaluate the code and class structure" part but you do need to do the rest of the proj00